src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -4,35 +4,28 @@
   <PropertyGroup>
```

Comment:
I'm fine if that is the case but I can definitely imagine dependencies that would make the .NET Core version run on .NET Framework. For example if this has a dependency on System.IO.FileSystem which doesn't ship inbox on .NET Framework. At a high level I want to make sure this task library will work both for MSBuild on .NET Core and MSBuild 14 and as long as that works I'm fine keeping just the one but if that doesn't work we might need to cross compile. @joperezr please test it in either case.